native/packages/react-native-bpk-component-carousel/src/BpkCarousel.js

```diff
@@ -107,26 +126,21 @@ class BpkCarousel extends React.Component<Props, State> {
     const childrenCount = React.Children.count(children);
 
     return (
-      <Fragment>
-        <ScrollView
-          accessible
-          accessibilityLabel={this.getAccessibilityLabel()}
-          onScroll={this.handleScroll}
-          scrollEventThrottle={SCROLL_EVENT_THROTTLE}
-          onLayout={this.onLayout}
-          horizontal
-          pagingEnabled
-          showsHorizontalScrollIndicator={false}
-          style={style}
-        >
-          {shouldRenderChildren &&
-            React.Children.map(children, child =>
-              React.cloneElement(child, {
-                ...child.props,
-                style: [child.props.style, { width, height }],
-              }),
-            )}
-        </ScrollView>
+      <View style={style} onLayout={this.onLayout}>
```

Comment:
I switched `Fragment` to `View` as `onLayout` isn't being called on the `RecyclerListView`, which we rely on. This was recently fixed, so when the next update is released we may be able to change this back: https://github.com/Flipkart/recyclerlistview/pull/193